sdk/search/search-documents/README.md

```diff
@@ -419,167 +416,6 @@ main();
 
 When retrieving results, a `facets` property will be available that will indicate the number of results that fall into each facet bucket. This can be used to drive refinement (e.g. issuing a follow-up search that filters on the `Rating` being greater than or equal to 3 and less than 4.)
 
-### Retrieve suggestions from an index
-
-If you [created a suggester](https://docs.microsoft.com/azure/search/index-add-suggesters) on your index, you can use it to return result suggestions for a user query.
-
-This example shows returning the top three suggestions for the input "wifi" from the suggester "sg":
-
-```js
-const { SearchClient, AzureKeyCredential } = require("@azure/search-documents");
-
-const client = new SearchClient(
-  "<endpoint>",
-  "<indexName>",
-  new AzureKeyCredential("<apiKey>")
-);
-
-async function main() {
-  const suggestResult = await client.suggest({
-    searchText: "wifi",
-    suggesterName: "sg",
-    select: ["HotelId", "HotelName"],
-    highlightPreTag: "<em>",
-    highlightPostTag: "</em>",
-    top: 3,
-  });
-
-  for (const result of suggestResult.results) {
-    console.log(`Suggestion: ${result.HotelName}; Match text: ${result.text}`);
-  }
-}
-
-main();
-```
-
-### Autocomplete a partial query using an index
-
-To implement type-ahead behavior in your application, you can query the index with partial user input and return a list of suggested completions. You must have [created a suggester](https://docs.microsoft.com/azure/search/index-add-suggesters) on your index first.
-
-The below example tries to complete the string "de" using the suggester named "sg" on the index:
-
-```js
-const { SearchClient, AzureKeyCredential } = require("@azure/search-documents");
-
-const client = new SearchClient(
-  "<endpoint>",
-  "<indexName>",
-  new AzureKeyCredential("<apiKey>")
-);
-
-async function main() {
-  const autocompleteResult = await client.autocomplete({
-    searchText: "de",
-    suggesterName: "sg",
-  });
-
-  for (const result of autocompleteResult.results || []) {
-    console.log(result.text);
-  }
-}
-
-main();
-```
-
-### Define a custom analyzer and test its output
```

Comment:
I hope some of these get turned into samples, since this one at least is non-trivial to make

Reply:
Yes. All of them becomes samples. That PR will be tonight... :-)